src/Sulu/Bundle/CustomUrlBundle/Resources/public/js/components/webspace/settings/custom-url/overlay/main.js

```diff
@@ -0,0 +1,408 @@
+                                $('#custom-url-target-value').val(title);
+                                $('#custom-url-target-button-clear').show();
+
+                                this.target = id;
+                                this.sandbox.emit('husky.overlay.custom-urls.slide-to', 0);
+                            }.bind(this),
+                            emitPreSelect: false
+                        }
+                    },
+                    {
+                        name: 'list-toolbar@suluadmin',
+                        options: {
+                            el: '#webspace-custom-urls-url-list-toolbar',
+                            hasSearch: false,
+                            instanceName: 'custom-urls-overlay',
+                            template: this.sandbox.sulu.buttons.get({
+                                delete: {
+                                    options: {disabled: true, callback: this.deleteUrl.bind(this)}
+                                }
+                            })
+                        }
+                    },
+                    {
+                        name: 'datagrid@husky',
+                        options: {
+                            el: '#webspace-custom-urls-url-list',
+                            instanceName: 'custom-urls-overlay',
+                            data: routeData,
+                            idKey: 'uuid',
+                            viewOptions: {
+                                table: {
+                                    selectItem: {
+                                        type: 'checkbox',
+                                        inFirstCell: false
+                                    }
+                                }
+                            },
+                            matchings: [
+                                {
+                                    content: this.translations.customUrl,
+                                    attribute: 'route'
+                                },
+                                {
+                                    content: this.translations.created,
+                                    attribute: 'created',
+                                    type: 'datetime'
+                                }
+                            ]
+                        }
+                    }
+                ]
+            );
+
+            if (!!this.data.baseDomain) {
+                $('#custom-url-input-container').show();
+            }
+
+            if (!!this.data.target) {
+                $('#custom-url-target-value').val(this.data.target.title);
+                $('#custom-url-target-button-clear').show();
+            }
+        },
+
+        /**
+         * Delete custom-url-route.
+         */
+        deleteUrl: function() {
+            var ids = this.sandbox.util.deepCopy($('#webspace-custom-urls-url-list').data('selected'));
+
+            // TODO how to handle overlays in overlays? error and question?
```

Comment:
i think i have talked to bernd about this but i cannot remember i will ask him on monday